src/components/designSystem/Filters/utils.ts

```
@@ -368,3 +369,13 @@
 }): string | null => {
   return searchParams.get(keyWithPrefix(key, prefix))
 }
+
+export const buildUrlForInvoicesWithFilters = (searchParams: URLSearchParams) => {
+  const searchParamsWithPrefix = searchParams.entries().reduce((acc, [key, value]) => {
+    const prefix = keyWithPrefix(key, INVOICE_LIST_FILTER_PREFIX)
+
+    return { ...acc, [prefix]: value }
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)